src/components/GalleryEditor/getInitialCollectionsFromServer.ts

```diff
@@ -125,3 +107,24 @@ export function getInitialCollectionsFromServer(
 
   return collections;
 }
+
+export function createEmptyCollection(): CollectionState {
+  const generatedCollectionId = generate12DigitId();
+  const generatedSectionId = generate12DigitId();
+
+  return {
+    dbid: generatedCollectionId,
+    localOnly: true,
+
+    liveDisplayTokenIds: new Set(),
+
+    name: '',
+    collectorsNote: '',
+    hidden: false,
+
+    sections: {
+      [generatedSectionId]: { columns: 3, items: [] },
+    },
+    activeSectionId: generatedSectionId,
+  };
+}
```

Comment:
![image](https://user-images.githubusercontent.com/12162433/216397210-b6203673-9d83-4ba3-b912-e9df4f8495f0.png)

should we also use this func here? in `GalleryEditorContext`